packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -75,40 +47,35 @@ class ZosmfApiCommon implements ZoweExplorerApi.ICommon {
         return this.session;
     }
 
+    private _getSession(serviceProfile: zowe.imperative.IProfileLoaded): zowe.imperative.Session {
```

Comment:
I noticed that this function is prefixed with an underscore, do we want to keep this or rename? 

Also checked around for other functions prefixed w/ underscore and didn't find them, but my regex test could have been off:
```regex
(?:private|public|(?:function|async function))\s_
```

Reply:
I extracted most of the `getSession` method that was duplicate code into the private `_getSession` method, because I was too lazy to think of a different name 😁  I'm open to following a different convention.